app/Models/EntryDAO.php

```diff
@@ -77,7 +83,7 @@ public function addEntry($valuesTmp) {
 			substr($valuesTmp['link'], 0, 1023),
 			$valuesTmp['date'],
 			time(),
-			hex2bin($valuesTmp['hash']),	// X'09AF' hexadecimal literals do not work with SQLite/PDO
+			$this->hasNativeHex() ? $valuesTmp['hash'] : pack('H*', $valuesTmp['hash']),	// X'09AF' hexadecimal literals do not work with SQLite/PDO	//hex2bin() is PHP5.4+
```

Comment:
I did not like using `hex2bin()` with MySQL anyway, because having binary data in the SQL logs was not nice, sometimes interfering with my SSH client.